### PR TITLE
some css changes, sidebar border

### PR DESCRIPTION
**Pull request by TKsM151**
- moved border on body to the sidebar instead for sidebarless pages to not look dumb
- center deal button text
- centered login and signup page forms

### CODE REVIEW

**Comment by TKsM151** [comment]
something for #164

File: templates/security/login.html.twig [hunk 1]
```diff
@@ -33,7 +33,8 @@
           {{- 'login_form.reset_password'|trans -}}
         </a>
       {% endif %}
+
+      <p>Don't have an account? <a href="">Sign up now.</a>
```
Comment (tstiegler) [inline templates/security/login.html.twig]: Does this need to link to the registration page?

Comment (tstiegler) [reply templates/security/login.html.twig]: Also, looks like a missing `</p>` at the end there.

**Comment by tstiegler** [comment]
Looks good!

**Pull request merged**